includes/templates/admin/dashboard.php

```diff
@@ -30,7 +30,11 @@
 }
 
 ?>
-
+<div id="stories-dashboard-no-js" class="wrap">
```

Comment:
Why not use the pre-existing `no-js` class here?

Reply:
See above